src/platform/plugins/shared/esql/public/triggers/esql_controls/control_flyout/identifier_control_form.test.tsx

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React from 'react';
+import { render, within, fireEvent } from '@testing-library/react';
+import { monaco } from '@kbn/monaco';
+import { dataPluginMock } from '@kbn/data-plugin/public/mocks';
+import { ESQLVariableType } from '@kbn/esql-validation-autocomplete';
+import { IdentifierControlForm } from './identifier_control_form';
+import { ESQLControlState, EsqlControlType } from '../types';
+
+jest.mock('@kbn/esql-utils', () => ({
+  getESQLQueryColumnsRaw: jest.fn().mockResolvedValue([{ name: 'column1' }, { name: 'column2' }]),
+}));
+
+describe('IdentifierControlForm', () => {
+  const dataMock = dataPluginMock.createStartContract();
+  const searchMock = dataMock.search.search;
+
```

Comment:
Shall we clear all mocks to be safe? 
```suggestion
  beforeEach(() => {
    jest.clearAllMocks();
  });
```
```

Reply:
Sure https://github.com/elastic/kibana/pull/210170/commits/7bb3bb14142baf4fbd9820d7feb558c6a292742a